### ISSUE TITLE
VSI/HSI Flags showing incorrectly 

### ISSUE THREAD

**Issue by JRMITCH86** [Kinkkujuustovoileipa/uh-60l#145]
**Describe the bug**
The CMD, NAV, GS flags are being displayed incorrectly.

**To Reproduce**
1. Ground tune a VOR and select BRG2 to VOR, then verify NO2 bearing pointer is receiving the station. 
2. Verify VOR/ILS is selected on the MODE SEL panel
3. Notice the NAV flag is still present 
4. Only after selecting a CIS MODE SEL are any navigation indications given

**Expected behavior**
CIS MODE SEL should not be required for "Raw data" to be displayed for a properly tuned VOR or ILS in the reception range. 

It's easier to explain the fact the CIS MODE SELECTOR is designed primarily to "Process" raw data that has been selected for navigation by the Pilot via the HSI/VSI MODE Selector, but its **not required.** Its primary function is to "aid" by providing steering cues in the form of the Pitch, Roll Command bars on the VSI as well as the Collective Position indicator where applicable.  

Take for example you wish to fly "shoot" an ILS approach with Raw Data only:
1. Once you properly tuned the ILS freq, 
2. Selected VOR/ILS on the HSI/VSI MODE SELECTOR, (This should cause the TO flag to point towards the station, NAV flag to retract as well as bring the course deviation bar "alive", usually full-scale deflection one side or the other based on the course.) 
3. Then set the inbound ILS course on your HSI 
You should have a bearing pointer to the ILS antenna location, Couse deviation bar. Once you are close enough to pick up the Glideslope Indicator that would be presented as well. Only the VSI GS flag would potentially be present if you had a "Loss or Unreliable signal". This was commonly referred to as shooting a "Raw Data" approach versus a "Processed Data" approach. 

Also, on the HSI/VSI MODE SELECTOR, the CRS HDG selection, only the PLT or CPLT can have command of the CISP at a time. Pressing this switch cycles one to the other and should be reflected on both sides of the cockpit to show which pilot has "Command" of the system. pg 3-70 

**Media**
**CMD** warning flag: On helicopters equipped with digital CIS processor the CMD flag will not come into view when the navigation signal becomes unreliable or lost (only power loss to the CISP should cause this flag to be visible, i.e. power lost or destroyed) pg 3-64
, 3.25.1.2
**ATT** warning flag: Indicates loss of vertical gyro power or VSI malfunction. pg 3-66
**NAV** warning flag: Indicates loss, or unreliable signal indication. pg 3-66
**GS** warning flag: Indicates loss, or unreliable signal indicator. pg 3-66

The specific functions of the CIS MODE SEL are:
HDG ON : Direct heading and roll signals to CIS processor for _steering commands_ that will allow the pilot to maintain a selected heading. pg 3-70

NAV ON: Gives _heading commands_ to acquire and track a selected VOR, ILS, DPLR, DPLR/GPS, or FM intercept, or to acquire and track glide slope beam. pg 3-70

ALT ON: Directs barometric pressure signals and _collective stick position_ signals to CIS processor. pg 3-70

**Additional context**
All references given are from TM 1-1520-237-10

Again hope this helps. Keep up the OUTSTANDING work!

**Comment by Kinkkujuustovoileipa** [comment]
Just wanted to say thank you to appreciate the effort you put into writing this bug report, this kind of information is extremely helpful for me :)

**Comment by Kinkkujuustovoileipa** [comment]
CISP logic overhauled and issues in this task fixed

**Comment by JRMITCH86** [comment]
> Just wanted to say thank you to appreciate the effort you put into writing this bug report, this kind of information is extremely helpful for me :)

No, thank you. I’ve been waiting for the day I could fly in DCS, one of the aircraft actually got to fly IRL is finally here. The vCSG3 is looking forward to using this mod on future msn’s.  Any information I can provide to make the aircraft as accurate as possible don’t hesitate to reach out.
